### ISSUE TITLE
✨ New card on homepage for IT training fund

### ISSUE THREAD

**Issue by NienkeBr** [GCTC-NTGC/gc-digital-talent#11799]
## ✨ Feature

We need a new card on the homepage under "Check it out" to link to the IT training fund page  #11794. 

## 🕵️ Details

The new card will come next to the two existing card under "Check it out" on the left.

The card will link to the new page on the IT training fund.

The copy will need to be confirmed before going live.

## 🎨 Design

For questions: Nienke

[See the design in the homepage Figma file.](https://www.figma.com/design/fqwWGkRDcwHY7Y1TFJC5oG/Homepage-(All-users)?node-id=412-1037&node-type=canvas&t=M0KabAf8pYOmmHgZ-0) **It's the design in version 07 with the 3 cards.**

## 📸 Screenshot

![image](https://github.com/user-attachments/assets/7e609439-fd89-446d-8f14-1969310f91b7)

## 🌎 Localization

Nienke will provide the French translation.

## ✅ Acceptance Criteria

- [ ] New card is added to the homepage check it out section

## 🛑 Blockers

```[tasklist]
### Blocked By
- [ ] Card should only be live once the static page is live as well.
```

**Comment by tristan-orourke** [comment]
@NienkeBr please attach the image file here

**Comment by petertgiles** [comment]
Image: https://github.com/GCTC-NTGC/gc-digital-talent/issues/11794#issuecomment-2427216865
